include/inexor/vulkan-renderer/application.hpp

```diff
@@ -33,6 +34,12 @@ class Application : public VulkanRenderer {
 
     std::unique_ptr<input::KeyboardMouseInputData> m_input_data;
 
+    // Let's just create 2 worlds for testing now.
+    // TODO: Remove this again.
+    std::array<std::shared_ptr<world::Cube>, 3> m_worlds;
+    std::vector<std::shared_ptr<world::Cube>> m_helper_box;
+    std::vector<glm::vec3> helper_box_positions;
```

Comment:
```suggestion
    std::vector<glm::vec3> m_helper_box_positions;
```

Reply:
- [ ] Oh I will remove the helper boxes again, we don't need them (see TODO there).